vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MessageTextItem.kt

```diff
@@ -80,17 +83,26 @@ abstract class MessageTextItem : AbsMessageItem<MessageTextItem.Holder>() {
                 it.bind(holder.messageView)
             }
         }
-        val textFuture = PrecomputedTextCompat.getTextFuture(
-                message ?: "",
-                TextViewCompat.getTextMetricsParams(holder.messageView),
-                null)
+        val textFuture = if (canUseTextFuture) {
```

Comment:
Why do we have 2 if(canUseTextFuture) ?

Reply:
One to compute it, one to apply it. I did not want to change method call order. I could test nullability of `textFuture` for the second one if you prefer, but it's the same